Server/include/OpcuaToolkitInfo.hpp.in

```diff
@@ -0,0 +1,32 @@
+/* © Copyright CERN, 2022.  All rights not expressly granted are reserved.
+ * OpcuaToolkitInfo.hpp
+ *
+ *  Created on: June 22, 2022
+ * 		Author: Paris Moschovakos <paris.moschovakos@cern.ch>
+ *
+ *  This file is part of Quasar.
+ *
+ *  Quasar is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public Licence as published by
+ *  the Free Software Foundation, either version 3 of the Licence.
+ *
+ *  Quasar is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public Licence for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with Quasar.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+namespace OpcuaToolkitInfo
+{
+
+std::string getOpcuaToolkitPath()
+{
+    return "@OPCUA_TOOLKIT_PATH@";
```

Comment:
Interesting method! If this turns out to be good, maybe the same thing should be done for Meta?

Reply:
It could be yes. It has the potential to simplify the chain metadata+python scripting+file gen+++ by simply adding a couple more of these methods.
For this PL I thought it is out-of-scope and that a separate ticket/effort/contrib can be dedicated for that. I also favor this idea but one should be ready to violate the ideology "if it does work, dont fix it" ;)